.npmrc

```diff
@@ -0,0 +1,2 @@
+tag-version-prefix=""
+message="chore(release): bumped version to %s"
```

Comment:
also newline, and actually I'm not sure if we need this (as we have semantic-release doing this for us)